frontend/src/scene/balance/BalanceView.js

```diff
@@ -87,6 +88,16 @@ const SubmitButton = styled(Button)`
     align-self: center;
 `;
 
+const ErrorMessage = styled('span')`
+    display: flex;
+    font-size: 2rem !important;
+    margin: 0 !important;
```

Comment:
!important is a style hack. Let's not use that

Reply:
This was copied from the existing code

Reply:
removed it